blocks/src/show-avatar/block.js

```diff
@@ -6,44 +6,44 @@
  */
 
 //  Import CSS.
-import './style.scss';
```

Comment:
LOTS of changes here. Mostly moving to import syntax and removing the items that have been define in block.json from the `registerBlockType` call